homeassistant/components/media_player/squeezebox.py

```diff
@@ -45,7 +45,7 @@
 
 SERVICE_CALL_METHOD = 'squeezebox_call_method'
 
-DATA_SQUEEZEBOX = 'squeexebox'
+DATA_SQUEEZEBOX = 'squeezebox'
```

Comment:
This seems to mainly be used to store internal data but this change might have unintended side-effects.